ebuild-writing/eapi/text.xml

```diff
@@ -616,6 +616,773 @@ installation targets.
+    </p>
+
+<codesample lang="ebuild">
+src_prepare() {
+	eapply -p0 "${FILESDIR}"/${P}-foo.patch
+	eapply_user
+}
+</codesample>
+
+  </dd>
+</dl>
+
+</body>
+</subsection>
+<subsection>
+<title>EAPI 8 commands</title>
+<body>
+
+<dl>
+  <dt>New econf-passed options</dt>
+  <dd>
+    <p>
+    The <c>econf</c> helper has been modified to pass two more options to
+    the configure script if the <c>--help</c> text indicates that they are
+    supported. These are:
+    </p>
+
+    <ul>
+      <li><c>--datarootdir="${EPREFIX}"/usr/share</c></li>
+      <li><c>--disable-static</c></li>
+    </ul>
+
+    <p>
+    The former option defines the base directory that is used to determine
+    locations for system/desktop-specific data files, e.g. .desktop files and
+    various kinds of documentation. This is necessary for ebuilds that override
+    <c>--prefix</c>, as the default path is relative to it.
+    </p>
+
+    <p>
+    The latter option disables building static libraries by default. This is
+    part of the ongoing effort to disable unconditional install of static
+    libraries
+    (<uri link="https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0302">
+    Gentoo Policy Guide, Installation of static libraries</uri>).
+    </p>
+  </dd>
+
+  <dt><c>dosym -r</c> to create relative symlinks</dt>
+  <dd>
+    <p>
+    Relative symlink targets tend to be more reliable. Consider the two
+    following examples:
+    </p>
+
+<codesample lang="ebuild">
+dosym "${EPREFIX}"/usr/lib/frobnicate/frobnicate /usr/bin/frobnicate
+dosym ../lib/frobnicate/frobnicate /usr/bin/frobnicate
+</codesample>
+
+    <p>
+    The first line creates a symlink using an absolute path. The problem with
+    that is if you mount your Gentoo system in a subdirectory of your root
+    filesystem (e.g. for recovery), the symlink will point at the wrong
+    location. Using relative symlinks (as demonstrated on the second line)
+    guarantees that the symlink will work independently of where the filesystem
+    is mounted.
+    </p>
+
+    <p>
+    There is also fact that you need to explicitly prepend <c>${EPREFIX}</c>
+    to the absolute paths passed as the first argument of <c>dosym</c>. Using
+    a relative target avoids the problem altogether and makes it less likely to
+    forget about the prefix.
+    </p>
+
+    <p>
+    However, in some instances determining the relative path could be hard or
+    inconvenient. This is especially the case if one (or both) of the paths
+    comes from an external tool. To make it easier, EAPI 8 adds a new <c>-r</c>
+    option that makes <c>dosym</c> create a relative symlink to the specified
+    path (similarly to <c>ln -r</c>):
+    </p>
+
+<codesample lang="ebuild">
+dosym -r /usr/lib/frobnicate/frobnicate /usr/bin/frobnicate
+</codesample>
+
+    <p>
+    Note that in this case, you do not pass <c>${EPREFIX}</c>. The helper
+    determines the *logical* relative path to the first argument and creates
```

Comment:
Don't we have some fancy tag instead of `*logical*`? ;-)

Reply:
Right, I had missed this one. We have `<e>` for this.